source/simulator/solver.cc

```diff
@@ -659,26 +659,26 @@ namespace aspect
     // create a completely distributed vector that will be used for
     // the scaled and denormalized solution and later used as a
     // starting guess for the linear solver
-    LinearAlgebra::BlockVector remap (introspection.index_sets.stokes_partitioning, mpi_communicator);
+    LinearAlgebra::BlockVector init_sol (introspection.index_sets.stokes_partitioning, mpi_communicator);
```

Comment:
same here -- can you spell out what `init_sol` should be? initial? initialized? I guess we just like long variable names that are self-descriptive.